cdc/model/changefeed.go

```diff
@@ -37,19 +37,34 @@ type ChangeFeedDetail struct {
 	AdminJobType AdminJobType    `json:"admin-job-type"`
 	Info         *ChangeFeedInfo `json:"-"`
 
-	filter              *filter.Filter
-	FilterCaseSensitive bool          `json:"filter-case-sensitive"`
-	FilterRules         *filter.Rules `json:"filter-rules"`
+	filter *filter.Filter
+	Config *ReplicaConfig `json:"config"`
```

Comment:
It seems like `ChangeFeedDetail` is the config itself. For example, it includes things like `SinkURI` and `TargetTs`. 

Maybe we should think about how we are going to configure change feeds later.

Reply:
More like an L2 config of a L1 config. We can refine it later.